python/tests/test_logging_config.py

```diff
@@ -0,0 +1,198 @@
+import logging
+import os
+from collections import namedtuple
+from pathlib import Path
+from unittest.mock import patch
+
+import pytest
+
+from open_discourse.helper_functions.logging_config import setup_and_get_logger
+
+# Definition Named Tuple for test cases, don't name it Testcase!!!
+CaseDataforTest = namedtuple("CaseDataforTest", ["input", "expected", "exception"])
+
+
+@pytest.fixture
```

Comment:
Glaube hier ist kein `unittest.mock` notwendig, das sollte einfacher mit pytest-mock (https://pytest-mock.readthedocs.io/en/latest/index.html) gehen. Gerne installieren bzw. als dependency hinzufügen, dann kannst du direkt `test(mocker)` verwenden, und in der Funktion dann den Patch direkt mit `mocker.patch()` ausführen

Reply:
Damit sparen wir uns eine ganze fixture Funktion